prow/config/config.go

```diff
@@ -216,14 +216,39 @@ type InRepoConfig struct {
 	AllowedClusters map[string][]string `json:"allowed_clusters,omitempty"`
 }
 
+func trimRepoPrefix(repo string) string {
+	repo = strings.TrimPrefix(repo, "http://")
+	repo = strings.TrimPrefix(repo, "https://")
+	return repo
+}
+
+func SplitRepoName(repo string) (string, string, error) {
+	// Normalize repo name to remove http:// or https://, this is the case for some
+	// of the gerrit instances.
+	repo = trimRepoPrefix(repo)
+	s := strings.SplitN(repo, "/", 2)
+	if len(s) != 2 {
+		return "", "", fmt.Errorf("repo %s cannot be split into org/repo", repo)
+	}
+	return s[0], s[1], nil
+}
+
 // InRepoConfigEnabled returns whether InRepoConfig is enabled for a given repository.
+// Assumes that config will not include http:// or https://
 func (c *Config) InRepoConfigEnabled(identifier string) bool {
-	if c.InRepoConfig.Enabled[identifier] != nil {
-		return *c.InRepoConfig.Enabled[identifier]
+	// SplitRepoName trims http:// and https:// from beginning of org name
+	split := false
+	org, _, err := SplitRepoName(identifier)
+	if err == nil {
+		split = true
 	}
-	identifierSlashSplit := strings.Split(identifier, "/")
-	if len(identifierSlashSplit) == 2 && c.InRepoConfig.Enabled[identifierSlashSplit[0]] != nil {
-		return *c.InRepoConfig.Enabled[identifierSlashSplit[0]]
+	normalizedIdentifier := trimRepoPrefix(identifier)
```

Comment:
Why not do this before the call to `SplitRepoName`, and then pass it along to `SplitRepoName`? That way you could avoid calling `trimRepoPrefix` twice.

Reply:
trimRepoPrefix is still required in SplitRepoName as that function is used elsewhere, so we will need both calls anyway.